test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala

```diff
@@ -160,6 +160,11 @@ class CheckAllAnswersAndSubmitControllerSpec
                   .fold(sys.error("Error"))(_.asInstanceOf[CompleteExemptionAndLossesAnswers]),
                 doc
               )
+              validateYearToDateLiabilityFirstReturnPage(
+                completeFillingOutReturn.draftReturn.yearToDateLiabilityAnswers
```

Comment:
`completeFillingOutReturn` derives its values from `completeReturn` - you should be able to just do `completeReturn.yearToDateLiabilityAnswers` without folding and sys.erroring. The same is true for the other answers

Reply:
done, thx